cartographer/mapping_3d/local_trajectory_builder.cc

```diff
@@ -195,13 +198,13 @@ const mapping::PoseEstimate& LocalTrajectoryBuilder::pose_estimate() const {
 
 std::unique_ptr<LocalTrajectoryBuilder::InsertionResult>
 LocalTrajectoryBuilder::InsertIntoSubmap(
-    const common::Time time, const sensor::RangeData& range_data_in_tracking,
+    const common::Time time, const sensor::RangeData& filtered_range_data,
```

Comment:
Revert? That it is filtered seems unimportant to understand what this function does, but that it still has to be transformed is important.

Reply:
I found it very unredable that the unfiltered variable in `AddAccumulatedRangeData` is also called `range_data_in_tracking`; and then `filtered_range_data` is passed again as `range_data_in_tracking` in `InsertIntoSubmap`. Maybe `filtered_range_data_in_tracking`?

Reply:
I'm not feeling strongly about this. I personally would always chose conciseness if in doubt, but that is just me. That not every name is instantly understood by everyone is unavoidable. More of a question of how big the problem is here.

Reply:
I think it's a subtle detail which should be made clearer, that filtered scans are inserted into submaps. If you don't mind, I'd like to keep it.